bucket/lua-windows.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
There already is a [lua-for-windows](https://github.com/ScoopInstaller/Main/blob/master/bucket/lua-for-windows.json). Is it correct to have both? Is lua-windows the correct name, given the name of this other package?

Reply:
`lua-for-windows` contains both lua5.1 and luarocks(may be 2.x), but the sources are not activate maintained.